testlib/src/test/java/com/diffplug/spotless/markdown/FlexmarkStepTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2016-2021 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless.markdown;
+
+import static org.junit.jupiter.api.condition.JRE.JAVA_14;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledForJreRange;
+
+import com.diffplug.spotless.StepHarness;
+import com.diffplug.spotless.TestProvisioner;
+
+@EnabledForJreRange(max = JAVA_14)
```

Comment:
Interesting max. Maybe a comment linking to something that tracks this limitation, so it's easy to fix when the time is right?

Reply:
This was also a copy mistake 😅 

I'll have a look if there are any restrictions. If not, I'll remove this